src/components/Header/Topic/Topic.tsx

```diff
@@ -73,21 +120,18 @@ export default function Topic() {
               <div className="w-[20%] p-6 text-left">
                 <ul>
                   {topics.map((topic, index) => (
-                    <li
+                    <button
```

Comment:
I think children of `<ul>` must be `<li>`?

https://stackoverflow.com/questions/65341792/element-a-not-allowed-as-child-of-element-ul-in-this-context-suppressing-furth

maybe wrap the buttons in li?